src/metabase/query_processor/expand.clj

```diff
@@ -181,6 +181,10 @@
 
 ;;; ## breakout & fields
 
+(s/defn ^:ql ^:always-validate binning-strategy :- FieldPlaceholder
```

Comment:
The strategy-name isn't used here. Right now I just have the automatic binning strategy so I didn't use it. My next steps here would be using that to figure out what kind of binning the user wanted. Values here would be regex, automatic, predefined etc.